src/modules/services/components/bulk/AssignServiceButton.tsx

```diff
@@ -77,23 +103,66 @@ const AssignServiceButton: React.FC<Props> = ({
     return 'Enroll + Assign';
   }, [client.activeEnrollment, isAssignedOnDate, localDisabled]);
 
+  const startIcon =
```

Comment:
I want to confirm the icon behavior that made the most sense to me:
- Check-mark icon takes priority, if the client is assigned on that date
- Only show the warning icon on alerts if check-mark is not shown

Reply:
Looks good to me! Thanks